Dockerfile

```diff
@@ -9,27 +20,50 @@ WORKDIR /home/node
 ENV NODE_ENV development
 
 COPY --chown=node:node package.json package-lock.json /home/node/
-RUN npm ci
+RUN npm ci --loglevel verbose
 
 COPY --chown=node:node . /home/node/
 RUN npm run build
 
 CMD ["node", "index.js"]
 
+# Pa11y
+FROM node:${NODE_VERSION}-alpine AS pa11y
```

Comment:
Good point. I think it's in here because we're copying in files from the development container but as we're no longer running pa11y and the app in the same container this is unnecessary. I'll move it to a separate Dockerfile.